garage/contrib/bichengcao/examples/trpo_gym_Acrobot-v1.py

```diff
@@ -2,25 +2,25 @@
 
 from garage.baselines import LinearFeatureBaseline
 from garage.envs import normalize
-from garage.envs.util import horizon, spec
 from garage.misc.instrument import run_experiment
 from garage.theano.algos import TRPO
+from garage.theano.envs import TheanoEnv
 from garage.theano.policies import CategoricalMLPPolicy
 
 
 def run_task(*_):
-    env = normalize(gym.make("Acrobot-v1"))
+    env = TheanoEnv(normalize(gym.make("Acrobot-v1")))
 
-    policy = CategoricalMLPPolicy(env_spec=spec(env), hidden_sizes=(32, 32))
+    policy = CategoricalMLPPolicy(env_spec=env.spec, hidden_sizes=(32, 32))
 
-    baseline = LinearFeatureBaseline(env_spec=spec(env))
+    baseline = LinearFeatureBaseline(env_spec=env.spec)
 
     algo = TRPO(
         env=env,
         policy=policy,
         baseline=baseline,
         batch_size=4000,
-        max_path_length=horizon(env),
+        max_path_length=env.horizon,
```

Comment:
since, in this example, we know that this env has a spec (from gym), you should instead call env.max_episode_steps

https://github.com/openai/gym/blob/master/gym/envs/registration.py#L64

Reply:
`horizon` will already call `env.spec.max_episode_steps` for the underlying env. This is because we wrapped env with TheanoEnv:
```
  File "garage/contrib/bichengcao/examples/trpo_gym_Acrobot-v1.py", line 23, in run_task
    max_path_length=env.max_episode_steps,
AttributeError: 'TheanoEnv' object has no attribute 'max_episode_steps'
```

Reply:
```python
if hasattr(env, "max_episode_steps") and not hasattr(TheanoEnv(env), "max_episode_steps"):
    raise SomethingIsWrongWithTheanoEnvException
```